From 3742c034164351881c98588f176212fa015021da Mon Sep 17 00:00:00 2001
From: Gerrit Pape <pape@smarden.org>
Date: Wed, 9 May 2007 08:36:16 +0000
Subject: [PATCH] bug#369742: pager.c: fallback to 'pager' not 'less' if $PAGER is unset.

---
 pager.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/pager.c b/pager.c
index 8bac9d9..50d9ec3 100644
--- a/pager.c
+++ b/pager.c
@@ -39,7 +39,7 @@ void setup_pager(void)
 	if (!pager)
 		pager = getenv("PAGER");
 	if (!pager)
-		pager = "less";
+		pager = "pager";
 	else if (!*pager || !strcmp(pager, "cat"))
 		return;
 
-- 
1.5.3.4

