From 97fe47516b4e8ef7e5f1eda568836200aef5676a Mon Sep 17 00:00:00 2001
From: Gerrit Pape <pape@smarden.org>
Date: Wed, 9 May 2007 08:36:16 +0000
Subject: [PATCH] bug#369742: pager.c: fallback to 'pager' not 'less' if $PAGER is unset.

http://bugs.debian.org/369742
---
 pager.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/pager.c b/pager.c
index ca002f9..1ee4566 100644
--- a/pager.c
+++ b/pager.c
@@ -39,7 +39,7 @@ void setup_pager(void)
 	if (!pager)
 		pager = getenv("PAGER");
 	if (!pager)
-		pager = "less";
+		pager = "pager";
 	else if (!*pager || !strcmp(pager, "cat"))
 		return;
 
-- 
1.5.4.4

