From 513c10130280f8ff18e5e3eb723488da3483762c Mon Sep 17 00:00:00 2001
From: Gerrit Pape <pape@smarden.org>
Date: Wed, 9 May 2007 08:36:16 +0000
Subject: [PATCH 1/5] bug#369742: pager.c: fallback to 'pager' not 'less' if $PAGER is unset.

http://bugs.debian.org/369742
---
 pager.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/pager.c b/pager.c
index 86facec..61ac7fc 100644
--- a/pager.c
+++ b/pager.c
@@ -58,7 +58,7 @@ void setup_pager(void)
 	if (!pager)
 		pager = getenv("PAGER");
 	if (!pager)
-		pager = "less";
+		pager = "pager";
 	else if (!*pager || !strcmp(pager, "cat"))
 		return;
 
-- 
1.6.5.2

